src/components/dailyQuestsPage/DailyQuests.jsx

```diff
@@ -0,0 +1,244 @@
+import React, { useEffect, useState } from "react";
+import { format, addDays } from "date-fns";
+import QuestModal from "./QuestModal";
+import NavbarTopLoggedIn from "../chooseNewResopage/NavbarTopLoggedIn";
+import SuggestedQuests from "../dailyQuestsPage/SuggestedQuests";
+import WeekDaysCarousel from "./WeekDaysCarousel";
+import QuestList from "./QuestList";
+import AddQuestButton from "./AddQuestButton";
+
+const daysOfWeek = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
+
+const normalizeFrequency = (freq) => {
+  if (!freq) return "";
+  const array = Array.isArray(freq)
+    ? freq
+    : freq.split(",").map((d) => d.trim());
+  return array.sort().join(", ");
+};
+
+const DailyQuests = () => {
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [selectedDay, setSelectedDay] = useState(currentDate);
+  const [quests, setQuests] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  const [showModal, setShowModal] = useState(false);
+  const [selectedQuest, setSelectedQuest] = useState(null);
+  const [title, setTitle] = useState("");
+  const [frequency, setFrequency] = useState("Daily");
+  const [icon, setIcon] = useState("+");
+
+  useEffect(() => {
+    const savedBookQuests =
+      JSON.parse(localStorage.getItem("readBooksQuests")) || [];
+
+    const defaultQuests = [
```

Comment:
extract it in separate file consts.js or config.js

Reply:
@Alina5884 these should be fetched from the backend.
We have a correspondent [API](https://ii-practicum-team-2-back.onrender.com/docs/#/DailyQuests) for daily quests.